client/src/EditorTab.jsx

```diff
@@ -75,7 +75,7 @@ export default function EditorTab() {
         value={codeSave ? codeSave.code : ""}
         onChange={preTextRemove}
         options={{
-          readOnly: !hasUser,
+          //readOnly: !hasUser,
```

Comment:
@gunjanbajaj1 - Please uncomment this line

Reply:
@xDAnkit Done